server/main.js

```diff
@@ -30,6 +29,7 @@ app.get('/', function(req, res) {
 // TODO: Error Handing here??
 
 // render everything that didnt get caught as index page
+// It might be a better idea to return a status 404 ??
```

Comment:
So the reason I did this is so that when a user tries to refresh a page in their browser while on a location like /app/orderbook - without this line, they get a page not found/error - so this just renders index instead (and angular should actually then render whatever their request is) (at least that is the intention)